tox.ini

```diff
@@ -16,4 +16,4 @@ commands = ./manage.py test
 
 [testenv:flake8]
 deps = flake8
-commands = flake8 invite tests setup.py
+commands = flake8  --ignore=E501 invite tests setup.py
```

Comment:
We could add a directive to the `[flake8]` section to ignore files in the migration dir. If we did that, we will still get a failure if the line length surpasses 99 chars in any other file.

Here is an example
https://github.com/unt-libraries/django-name/blob/master/tox.ini#L6-L7